src/Router.php

```diff
@@ -126,6 +122,12 @@ public function route()
         $requestUri = parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH);
         $requestUri = preg_replace("|^$base/?|", '', $requestUri);
         $requestUri = ltrim($requestUri, '/'); //ensure left-leading "/" is stripped.
+	    $isJsonRequest =  strpos($requestUri, 'wp-json') === 0 ;
+
+	    if (is_admin() || !defined('WP_USE_THEMES') || $isJsonRequest) {
```

Comment:
the first two clauses are redundant, as this function won't be called at all if true